packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem/AccountItemContent.tsx

```diff
@@ -91,12 +93,16 @@ export const AccountItemContent = ({
                 </AccountLabelContainer>
                 {customFiatValue && !isTestnet(symbol) ? (
                     <HiddenPlaceholder>
-                        <FiatAmountFormatter
-                            value={customFiatValue}
-                            currency={localCurrency}
-                            minimumFractionDigits={0}
-                            maximumFractionDigits={0}
-                        />
+                        {valueLoading ? (
+                            <SkeletonRectangle animate={shouldAnimate} />
```

Comment:
FiatAmountFormatter has property `showLoadingSkeleton`

Reply:
Aha, I dont' think so. `FiatAmountFormatter` doesn't have that but `<FiatValue>` does.